scripts/install-python.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -x
+
+# Install Python
+yum -y install deltarpm epel-release
+yum -y install python34
```

Comment:
Single concern here: GeoPySpark relies on Python 3.5.  As I recall.  Might want to save us the trouble in the future and go directly to that version.